app/config/tests/test_api.py

```diff
@@ -0,0 +1,102 @@
+from config.api import root
+from ninja import NinjaAPI
+from ninja.errors import AuthenticationError
+from ninja.errors import HttpError
+from ninja.errors import ValidationError as NinjaValidationError
+from utils.testing import TestClient
+
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.http import Http404
+from django.http import HttpRequest
+from django.test import TestCase
+
+
+class ApiTestCase(TestCase):
+
+    def test_checker_url(self):
```

Comment:
I think this one is semantically wrong here. The checker isn't part of the API.

Reply:
Will split to two files, one for the checker and one for the error handlers.